gatsby/util/extract-personio-slug.ts

```diff
@@ -1,21 +0,0 @@
-export const PERSONIO_SLUG_FIELD_NAME = 'Slug';
-
-/**
```

Comment:
I found no usage of this file and therefore deleted it. Also deleted is the `sy-personio-jobs-source-plugin`

Reply:
yes please, we don't care for Personio anymore 😌